frontend/packages/design-system/src/themes/Input.ts

```diff
@@ -0,0 +1,37 @@
+import textTheme from './Text';
+
+const disabledStyle = {
+  borderColor: 'neutral.200',
+  backgroundColor: 'neutral.100',
+  color: 'neutral.200',
+};
+
+const activeStyle = {
+  borderColor: 'primary.500',
+  backgroundColor: 'secondary.100',
+  color: 'neutral.500',
+};
+
+export default {
+  baseStyle: {
+    backgroundColor: 'white',
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderColor: 'neutral.300',
+    borderRadius: 'nrc_xs',
+    color: 'neutral.300',
+    padding: 3,
+    _web: {
+      ...textTheme.variants.bodyText,
+    },
```

Comment:
Does this get applied to ios/android if it's under web? Should it not live at the top level?

Reply:
I think it would not get applied to ios/android. I'll add it back then?
I would personally also have put it at top level, but this value lives there in the default theming, and if I want to overwrite it, I need to put it here.

Reply:
> this value lives there in the default theming
I can't see this in the default theming (https://github.com/GeekyAnts/NativeBase/blob/master/src/theme/components/input.ts)

Does it not work at the top level?

Reply:
it didn't when I tried